src/main/scala/zio/kafka/client/Consumer.scala

```diff
@@ -13,22 +13,71 @@ import zio.stream._
 
 import scala.jdk.CollectionConverters._, scala.collection.compat._
 
+trait KafkaConsumer {
```

Comment:
Let's call this `Consumer`.

```suggestion
trait Consumer {
```